palette_derive/Cargo.toml

```diff
@@ -14,7 +14,7 @@ license = "MIT OR Apache-2.0"
 proc-macro = true
 
 [dependencies]
-syn = { version = "0.14", features = ["visit", "extra-traits"] }
+syn = { version = "0.15.14", features = ["extra-traits"] }
```

Comment:
I'm usually accepting patches, so I would have set it to just 0.15.

Reply:
Done.